### DISCUSSION TITLE
Session drop

### DISCUSSION THREAD

**Discussion by TJ-BitCanna** [BitCannaGlobal/WeedMobile#89] [General]
As reported by @alibabaedge:
The wallet session drops, when the first time give permissions to use camera.

And also on iOS there's no way to choose "permit this time only".

**Comment by RaulBernal** [comment]
Yeah, is "normal" (could be improved) behavior when you change a System param to be kicked from the App
It's also related with this other issue (where the SDK that interact with iOS should be fixed and apply by us when is done)
https://github.com/BitCannaGlobal/WeedMobile/discussions/86

**Reply by TJ-BitCanna** [reply]
No, on other apps (like Binance, Bybit, etc) the app does not close when the param is set for the _**first**_ time.

It is normal behaviour when the system parameter is changed by the user in the Settings screen of the phone itself.

So this remains an issue for setting the **_initial_** rights for the application.
